licensecheck.properties

```diff
@@ -0,0 +1,25 @@
+category=Languages
```

Comment:
You haven't listed a `homepageUrl`. That's how users in commercial versions will find your binaries for download

Reply:
Just noticed this value. Category should be 'External Analyzers' since you're not covering a new language.